Src/Web/Web/Web.csproj

```diff
@@ -0,0 +1,65 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\..\Common.props" />
+  <Import Project="..\..\..\Nupkg.props" />
+
+  <PropertyGroup>
+    <RootNamespace>Microsoft.ApplicationInsights.Web</RootNamespace>
+    <AssemblyName>Microsoft.AI.Web</AssemblyName>
+    <DocumentationFile>$(OutputPath)\$(AssemblyName).XML</DocumentationFile>
+    <TargetFramework>net45</TargetFramework>
+    <Prefer32Bit>false</Prefer32Bit>
+    <DefineConstants>$(DefineConstants);NET45;ALLOW_AGGRESSIVE_INLIGNING_ATTRIBUTE</DefineConstants>
```

Comment:
no need to define `NET45` in new project types btw

Reply:
I think it's best to leave it for now unless we want to investigate the full impact.
Our Common projects are still using the NET45 preprocessor. 
In the future, we could change our preprocessors to be NET45 by default and the if condition could be NETSTANDARD.